src/styles/blocks/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  padding-left: 50px;
+  padding-right: 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 1px 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+}
+
+.nav {
+  &__list {
```

Comment:
create separated file for nav BEM block

Reply:
looks like this comment not fixed
https://github.com/mate-academy/layout_catalog/pull/3124/files#r1191160534 or your changes not pushed